info/container.go

```diff
@@ -197,6 +197,8 @@ type CpuStats struct {
 		// Unit: nanoseconds
 		System uint64 `json:"system"`
 	} `json:"usage"`
+	// Smoothed average of number of runnable threads x 1000.
+	// We multiply by thousand to avoid using floats, but preserving precision.
 	Load int32 `json:"load"`
```

Comment:
We smooth it over last 10s. If the load changes from 2 to 50 and stays there, this load value should get to 50 within 10s. The instantaneous value is very spiky and not useful for analysis. If a tool wants to do its own calculation or look for bursts, the value is still available in loadstats.NrRunning.

I can change the field to LoadAverage and add the 10s interval in comments.